### PR TITLE
Sync perltidy with openQA#773f59b

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
needs updates to test repo, too.

File: backend/qemu.pm [hunk 1]
```diff
@@ -275,7 +276,10 @@ sub start_qemu {
 
     my $qemubin = $ENV{QEMU};
     unless ($qemubin) {
-        my @candidates = $vars->{QEMU} ? ('qemu-system-' . $vars->{QEMU}) : qw(kvm qemu-kvm qemu qemu-system-x86_64 qemu-system-ppc64);
+        my @candidates
```
Comment (coolo) [inline backend/qemu.pm]: this should really be written differently

File: basetest.pm [hunk 2]
```diff
@@ -528,7 +534,12 @@ sub verify_sound_image {
...
         $self->record_screenmatch($img, $foundneedle, [$mustmatch], $rsp->{candidates});
         my $lastarea = $foundneedle->{area}->[-1];
-        bmwqemu::fctres(sprintf("found %s, similarity %.2f @ %d/%d", $foundneedle->{needle}->{name}, $lastarea->{similarity}, $lastarea->{x}, $lastarea->{y}));
+        bmwqemu::fctres(
+            sprintf(
+                "found %s, similarity %.2f @ %d/%d",
+                $foundneedle->{needle}->{name},
+                $lastarea->{similarity},
+                $lastarea->{x}, $lastarea->{y}));
         return $foundneedle;
```
Comment (coolo) [inline basetest.pm]: hmm, did you work on this manually or why is perltidy once putting )) on an extra line and once not?

File: consoles/sshVirtsh.pm [hunk 3]
```diff
@@ -144,7 +144,9 @@ sub _init_xml {
         }
         if (!get_var('BIOS')) {
             # We know this won't go well.
-            die "No UEFI firmware can be found on hypervisor " . get_var('VIRSH_HOSTNAME') . "\n. Please specify BIOS or UEFI_BIOS or install an appropriate package.";
+            die "No UEFI firmware can be found on hypervisor "
+              . get_var('VIRSH_HOSTNAME')
+              . "\n. Please specify BIOS or UEFI_BIOS or install an appropriate package.";
```
Comment (coolo) [inline consoles/sshVirtsh.pm]: I suggest to move the \n one up - this way looks really strange

File: consoles/sshVirtsh.pm [hunk 4]
```diff
@@ -311,7 +313,10 @@ sub add_disk {
                 my $chan             = $self->{sshVMwareServer}->channel();
                 # Power VM off, delete it's disk image, and create it again.
                 # Than wait for some time for the VM to *really* turn off.
-                $chan->exec("vmid=\$(vim-cmd vmsvc/getallvms | awk '/ " . $self->name . " / { print \$1 }'); vim-cmd vmsvc/power.getstate \$vmid; vim-cmd vmsvc/power.off \$vmid; vim-cmd vmsvc/power.getstate \$vmid; vmkfstools -v1 -U $vmware_disk_path; vmkfstools -v1 -c $size --diskformat thin $vmware_disk_path; sleep 10");
+                $chan->exec("vmid=\$(vim-cmd vmsvc/getallvms | awk '/ "
+                      . $self->name
+                      . " / { print \$1 }'); vim-cmd vmsvc/power.getstate \$vmid; vim-cmd vmsvc/power.off \$vmid; vim-cmd vmsvc/power.getstate \$vmid; vmkfstools -v1 -U $vmware_disk_path; vmkfstools -v1 -c $size --diskformat thin $vmware_disk_path; sleep 10"
```
Comment (coolo) [inline consoles/sshVirtsh.pm]: I guess this would benefit from a temporary variable

File: mmapi.pm [hunk 5]
```diff
@@ -20,7 +20,8 @@ use strict;
 use warnings;
 
 use base 'Exporter';
-our @EXPORT = qw(get_children_by_state get_children get_parents get_job_info get_job_autoinst_url get_job_autoinst_vars wait_for_children wait_for_children_to_start api_call);
+our @EXPORT
+  = qw(get_children_by_state get_children get_parents get_job_info get_job_autoinst_url get_job_autoinst_vars wait_for_children wait_for_children_to_start api_call);
```
Comment (coolo) [inline mmapi.pm]: please split this array

**Comment by okurz** [comment]
so far I did not do any manual tuning. I will try to make it look nice then.

**Comment by coolo** [comment]
So how is your progress?

**Comment by okurz** [comment]
none at all

**Pull request merged**